### ISSUE TITLE
Kyro Class is not registered: com.digitalpebble.stormcrawler.persistence.Status

### ISSUE THREAD

**Issue by fackse** [DigitalPebble/storm-crawler#926]
In version 1.17 the following error message appears in the Storm UI: 
` java.lang.RuntimeException:  java.lang.IllegalArgumentException: Class is not registered:  com.digitalpebble.stormcrawler.persistence.Status Note: To register this class use: kryo.register(com.digitalpe....`
The following components output this error:
1. memoryspout
2. fetch

We are using storm-crawler/external/elasticsearch/

Maybe this is related to #924. Attached is the used pom.xml (Githib doesn't like xml files, so I had to change the extension to .txt)

Let me know if you need more information.

Thanks and kind regards
Norman
[pom.txt](https://github.com/DigitalPebble/storm-crawler/files/7548315/pom.txt)

**Comment by jnioche** [comment]
thanks @fackse, I think this is a duplicate of #924
not sure why we are seeing these exceptions only now but I will fix it shortly
please note that the 1.x branch is not active, the fix will be in main branch. You will have to move to Storm2 in order to benefit from it

**Comment by jnioche** [comment]
Have fixed #924
@fackse the fix does not need you to use 2.x, it should be a matter of adding 

```
  topology.kryo.register:
    - com.digitalpebble.stormcrawler.Metadata
    - com.digitalpebble.stormcrawler.persistence.Status

```

to the configuration of your topology.
Please let us know if that doesn't fix it
